Food/go.lua

```diff
@@ -1,5 +1,5 @@
 local name = "go"
-local version = "1.15.8"
+local version = "1.16"
```

Comment:
Unfortunately, Go itself doesn't like semver if it ends in a `0` 🤔

Reply:
Hmm... Prior versions of Go accepted the `.0`. Guess they finally removed it, though for what reason I can't tell. Yay for not following semantic versioning?

A common pattern used in other formulas is the following:

```
local release = "1.16"
local version = release .. ".0"
```

If you update all the `url` fields to reference `release` instead of `version`, it should work as expected.